msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1500,58 +1498,58 @@ private AccountRecord selectAccountRecordForTokenRequest(
             // If the account is named like <tenant_name>.onmicrosoft.com we need to query the OpenId
             // Provider Configuration Metadata in order to get the tenant id. Once we have the
             // tenant id, we must then select the appropriate home or profile.
-            String tenantIdNameOrAlias = aadAuthority.getAudience().getTenantId();
+            String tenantId = aadAuthority.getAudience().getTenantId();
 
             // The AccountRecord we'll use to request a token...
             final AccountRecord accountRecord = new AccountRecord();
             accountRecord.setEnvironment(multiTenantAccount.getEnvironment());
             accountRecord.setHomeAccountId(multiTenantAccount.getHomeAccountId());
 
-            final boolean isUuid = isUuid(tenantIdNameOrAlias);
+            final boolean isUuid = isUuid(tenantId);
```

Comment:
nit: StringUtil.isUuid?

Reply:
it's the same method, i declared the static method as import.